main.py

```diff
@@ -84,15 +85,23 @@ class BlammoBot(BaseBot):
     global _check_all_alphanumeric
     global _reload_trivia
     global _reload_scramble
+    global _reload_escramble
     global _reload_secretcommand
 
     global scramble  # trivia equivalent: trivia
     global scramble_word  # trivia equivalent: questions  (scrambled, unscrambled)
     global scramble_started  # trivia equivalent: trivia_started
 
+    global emote_scramble
+    global emote_scramble_started
+    global emote_scramble_word
+
     questions = None
+    scramble_word = None
```

Comment:
Added this because running `[prefix]dumptologs` from a fresh bot session crashes the `dumptologs` command due to `scramble_word` being undefined.